$NetBSD$

Don't buy into the idea that a 64-bit machine somehow implies using
libjpeg-turbo instead of ordinary libjpeg.

--- CMakeLists.txt.orig	2022-11-11 20:56:43.272205371 -0800
+++ CMakeLists.txt	2022-11-11 20:57:02.925939237 -0800
@@ -479,11 +479,6 @@
     endif()
     # Can be 'libmypaint' or 'libmypaint-1.x'
     pkg_search_module(MYPAINT_LIB REQUIRED libmypaint libmypaint-1.3>=1.3)
-    
-    if(PLATFORM EQUAL 64)
-        find_library(TURBOJPEG_LIB turbojpeg)
-        message("**************** turbojpeg lib:" ${TURBOJPEG_LIB})
-    endif()
 endif()
 
 
